packages/components/package.json

```diff
@@ -80,6 +80,7 @@
         "embla-carousel-react": "8.0.0-rc12",
         "gh-pages": "^2.1.1",
         "glob": "^7.1.5",
+        "lodash.debounce": "^4.0.8",
```

Comment:
since you've added a new dependency here you should run `npm run bootstrap:dev` in order to update package-lock.json of root.
then if there's any change in the package-lock.json please commit that too.